HOW_TO.md

```diff
@@ -103,22 +107,113 @@ any template .yaml files.
+5. Create new `workload method` for Pod and VM under `BenchmarkOperatorWorkloads class` section in [benchmark_operator_workloads.py](benchmark_runner/benchmark_operator/benchmark_operator_workloads.py).
    It can be duplicated from existing workload method: `def stressng_pod` or `def stressng_vm` and customized workload run steps accordingly
-6. Add workload method name (workload_pod/workload_vm) to environment_variables_dict['workloads'] in [environment_variables.py](benchmark_runner/main/environment_variables.py)
-7. Create workload folder in the [templates](benchmark_runner/common/template_operations/templates) directory.  Create the following files in that directory:
-   1. workload_data_template for configuration parameters, e. g. [stressng_data_template.yaml](benchmark_runner/common/template_operations/templates/stressng/stressng_data_template.yaml).  The data template is structured as discussed [below](#data-template).
-   2. workload pod and VM custom resource template inside [internal_data](benchmark_runner/common/template_operations/templates/stressng/internal_data)
-8. Add workload folder path in [MANIFEST.in](MANIFEST.in), add 2 paths: the workload path to 'workload_data_template.yaml' and path to 'internal_data' Pod and VM template yaml files
-9. Add tests for all new methods you write under `tests/integration`.
-10. Update the golden unit test files as described [above](#add-new-workload-modify-parameters-to-workload-or-change-parameters-for-any-ci-job)
-11. For test and debug workload, need to configure [environment_variables.py](benchmark_runner/main/environment_variables.py)
-   1. Fill parameters: workload, kubeadmin_password, pin_node_benchmark_operator, pin_node1, pin_node2, elasticsearch, elasticsearch_port
-   2. Run [main.py](/benchmark_runner/main/main.py)  and verify that the workload run correctly
-   3. The workload can be monitored and checked through 'current run' folder inside the run workload flavor (default flavor: 'test_ci')
-12. Open Kibana url and verify workload index populate with data:
-   1. Create the workload index: Kibana -> Hamburger tab -> Stack Management -> Index patterns -> Create index pattern -> workload-results -> timestamp -> Done
-   2. Verify workload-results index is populated: Kibana -> Hamburger tab -> Discover -> workload-results (index) -> verify that there is a new data
+6. Create dedicate `workload class` StressngPod or StressngVM in dedicate module `stressng_pod.py` or `stressng_vm.py` and customized workload run steps accordingly [stressng_pod.py](benchmark_runner/benchmark_operator/stressng_pod.py) 
+7. Add workload method name (workload_pod/workload_vm) to environment_variables_dict['workloads'] in [environment_variables.py](benchmark_runner/main/environment_variables.py)
+8. Create workload folder in the [templates](benchmark_runner/common/template_operations/templates) directory.  Create the following files in that directory:
```

Comment:
That is useful because its a link to the folder

Reply:
Then at least write out the full name in the text:

```
[benchmark_runner/benchmark_operator/stressng_pod.py](benchmark_runner/benchmark_operator/stressng_pod.py)
```
I don't know whether or not the link works inside an IDE, but it will not work if I'm simply trying to read the file.

Reply:
Fixed for all paths